apps/server/src/infra/etherpad-client/etherpad-client.adapter.ts

```diff
@@ -221,6 +246,63 @@ export class EtherpadClientAdapter {
 		}
 	}
 
+	public async listAuthorsOfPad(padId: EntityId): Promise<AuthorId[]> {
+		const response = await this.tryGetAuthorsOfPad(padId);
+		const authors = this.handleEtherpadResponse<InlineResponse20013>(response, { padId });
+
+		if (!this.isObject(authors)) {
+			throw new InternalServerErrorException('Etherpad listAuthorsOfPad response is not an object');
+		}
+
+		const authorIds = authors?.authorIDs as AuthorId[];
+
+		return authorIds;
+	}
+
+	private async tryGetAuthorsOfPad(padId: string): Promise<AxiosResponse<InlineResponse20013>> {
+		try {
+			const response = await this.padApi.listAuthorsOfPadUsingGET(padId);
+
+			return response;
+		} catch (error) {
+			throw EtherpadResponseMapper.mapResponseToException(EtherpadErrorType.CONNECTION_ERROR, { padId }, error);
+		}
+	}
+
+	public async deleteSession(sessionId: EntityId): Promise<InlineResponse2001 | undefined> {
+		const response = await this.tryDeleteSession(sessionId);
+		const responseData = this.handleEtherpadResponse<InlineResponse2001>(response, { sessionId });
+
+		return responseData;
+	}
+
+	private async tryDeleteSession(sessionId: string): Promise<AxiosResponse<InlineResponse2001>> {
+		try {
+			const response = await this.sessionApi.deleteSessionUsingPOST(sessionId);
+
+			return response;
+		} catch (error) {
+			throw EtherpadResponseMapper.mapResponseToException(EtherpadErrorType.CONNECTION_ERROR, { sessionId }, error);
+		}
+	}
+
+	public async deletePad(padId: EntityId): Promise<InlineResponse2001 | undefined> {
+		const response = await this.tryDeletePad(padId);
+		const responseData = this.handleEtherpadResponse<InlineResponse2001>(response, { padId });
+
+		return responseData;
+	}
+
+	private async tryDeletePad(sessionId: string): Promise<AxiosResponse<InlineResponse2001>> {
```

Comment:
Incorrect parameter that expects a padId

Reply:
fixed